internal/broker/bindings/bindings_manager.go

```diff
@@ -73,13 +75,12 @@ func (c *ServiceAccountBindingsManager) Create(ctx context.Context, instance *in
 		return "", time.Time{}, fmt.Errorf("while creating a service account: %v", err)
 	}
 
-	_, err = clientset.RbacV1().ClusterRoles().Create(ctx,
+	_, e := rbacClient.ClusterRoles().Create(ctx,
```

Comment:
What is the reason that we change the name from "err" to "e"?

Reply:
I was debugging and forgot to change it back, reverting...